### PR TITLE
Avatar Component

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review my avatar component page

Live link: https://css-companion.netlify.app/components/component.html

this page has

- nav element: Brand name and links to the home page and components docs
- main: Contains the documentation of avatar component with snippets and examples of associated avatar
- aside: contains all the components links
- footer: contains links to the various social sites

image
image

### CODE REVIEW

File: component/avatar/component.css [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+
+main{
+    
+    /* grid-template-areas: 200px 1fr; */
+    /* align-items: center; 
+    padding: 3.5rem;
+    text-align: center;
+    margin-left: 1rem;
+    margin-right: 1rem;
+    margin: auto; */
```
Comment (samflab) [inline component/avatar/component.css]: remove all the commented code

File: component/avatar/component.css [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+    font-size: 1.2rem;
+    padding-left: 7rem;
+    padding-right: 7rem;
+
+    
+   
+}
+
+main p{
+    letter-spacing: 0.1rem;
```
Comment (samflab) [inline component/avatar/component.css:99]: for values that are this small, you can use `px` instead.

File: component/avatar/component.css [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+    color: var(--secondary-color);
+}
+
+iframe{
+    padding-top: 1rem;
+}
+
+
+
+
```
Comment (samflab) [inline component/avatar/component.css:155]: you can remove extra space from here

File: component/avatar/component.css [hunk 4]
```diff
@@ -0,0 +1,542 @@
...
+} 
+
+
+ 
+
+
+   
+
```
Comment (samflab) [inline component/avatar/component.css:199]: here as well, remove extra space

**Review by samflab — APPROVED** [review]

File: component/avatar/component.css [hunk 5]
```diff
@@ -0,0 +1,542 @@
...
+   "footer footer";
+}
+
+nav{
+    display: flex;
+    gap: 20px;
+    width: 100vw;
+    flex-direction: column;
+    grid-area: nav;
+
```
Comment (khushijohri2001) [inline component/avatar/component.css]: You can remove these extra spaces here

File: component/avatar/component.css [hunk 6]
```diff
@@ -0,0 +1,542 @@
...
+nav h2{
+    position: relative;
+    top: 35px;
+    font-size: 2rem;
+    line-height: 2.5rem;
+    letter-spacing: 0.125rem;
+}
+
+.imp-text{
+    font-family: 'Redressed', cursive;
```
Comment (khushijohri2001) [inline component/avatar/component.css]: here too

File: component/avatar/component.css [hunk 7]
```diff
@@ -0,0 +1,542 @@
...
+    color: var(--primary-color);
+    
+}
+
+a:hover{
+    color: #f472b6;
+  
+}
+
+main{
```
Comment (khushijohri2001) [inline component/avatar/component.css:90]: You can remove these comments if no needed.

**Review by khushijohri2001 — APPROVED** [review]

**Pull request merged**